sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShuffleBufferCatalog.scala

```diff
@@ -209,7 +205,7 @@ class ShuffleBufferCatalog(
    */
   def acquireBuffer(tableId: Int): RapidsBuffer = {
     val shuffleBufferId = getShuffleBufferId(tableId)
-    acquireBuffer(shuffleBufferId)
+    acquireBuffer(shuffleBufferId, unspill = true)
```

Comment:
I would expect this to be false by default.

Reply:
No longer needed.